### ISSUE TITLE
proposal. free tx of 64 bytes 

### ISSUE THREAD

**Issue by zhen-kuo** [PandoraPay/go-pandora-pay#14]
https://github.com/PandoraPay/go-pandora-pay/blob/0eb16dcd9c20e71a92ac4132a40dcffbc4c1e167/blockchain/transactions/transaction/transaction_zether/transaction_zether_payload/transaction_zether_payload.go#L38-L39

https://github.com/PandoraPay/go-pandora-pay/blob/0eb16dcd9c20e71a92ac4132a40dcffbc4c1e167/blockchain/transactions/transaction/transaction_zether/transaction_zether_payload/transaction_zether_payload.go#L396-L401

instead of whisper bytes you could encode the amount in the encrypted data ( message )

**Comment by zhen-kuo** [comment]
new message data = encrypt( encode( uvarint, rest of data) )

**Comment by zhen-kuo** [comment]
maybe encrypted message is enforced by default. it is better for privacy. users might opt for a bigger plain text message (or attestation).

**Comment by ibudisteanu** [comment]
Let me understand your suggestion. You basically say that the encrypted message should exist by default. It should be even if no message is encrypted by the user. This would eat 32 bytes. The balance can be encoded into the encrypted message. Overall your suggestion would reduce the tx with only 32 bytes.

**Comment by zhen-kuo** [comment]
yes !! this way transactions are more private. encrypted data by default. yes this scheme saves only 32 bytes.

**Comment by ibudisteanu** [comment]
It is indeed an interesting proposal. But some work needs to be done for the encoding schema.
